models/covidCrushers/analysis of model.py

```diff
@@ -0,0 +1,141 @@
+
+
+from math import sqrt
+from multiprocessing import cpu_count
+from joblib import Parallel
+from joblib import delayed
+from warnings import catch_warnings
+from warnings import filterwarnings
+from statsmodels.tsa.holtwinters import ExponentialSmoothing
+from sklearn.metrics import mean_squared_error
+from pandas import read_csv
+from numpy import array
+import numpy as np
+
+fips_library = [42039, 42079, 42077, 42083, 42089, 42027, 42041, 42043, 42045, 42051, 42073, 42081, 42091, 42093, 42095, 42047, 42049, 42075, 42007, 42055, 42057, 42071, 42085, 42059, 42061, 42063, 42065, 42067, 42069, 42087, 42097, 42099, 42101, 42103, 42105, 42107, 42109, 42111, 42113, 42115, 42117, 42119, 42121, 42123, 42125, 42127, 42129, 42131, 42133, 42019, 42021, 42023, 42025, 42029, 42031, 42033, 42035, 42037, 42053, 42001, 42003, 42005, 42009, 42011, 42013, 42015, 42017]
+
+# one-step Holt Winter’s Exponential Smoothing forecast
+def exp_smoothing_forecast(history, config):
+    t,d,s,p,b,r = config
+    # define model
+    history = array(history)
+    model = ExponentialSmoothing(history, trend=t, damped=d, seasonal=s, seasonal_periods=p)
+    # fit model
+    model_fit = model.fit(optimized=True, use_boxcox=b, remove_bias=r)
+    # make one step forecast
+    yhat = model_fit.predict(len(history), len(history))
+    return yhat[0]
+
+# root mean squared error or rmse
+def measure_rmse(actual, predicted):
+    return sqrt(mean_squared_error(actual, predicted))
+
+# split a univariate dataset into train/test sets
+def train_test_split(data, n_test):

```

Comment:
(train_test_split function) - take a look at the KFold function in sklearn.model_selection library